database/src/seeds/03_dwc_spatial_transform.ts

```diff
@@ -102,8 +103,8 @@ const transformString = `
     ),
     location_coord as (
         select
-            st_x(pt) x,
-            st_y(pt) y,
+            coalesce(st_x(pt), 0) x,
+            coalesce(st_y(pt), 0) y,
```

Comment:
This is so that records with no spatial coordinates produce a [0, 0] coordinates array, rather than [null, null] (which isn't a valid coordinate array and breaks the geojson openapi spec).

Ideally, we don't get records with no spatial coordinates as they are caught by SIMS first, but this way at least they won't break biohub.